### PR TITLE
ER-349 Displaying QA reviews on dashboard

### PR DESCRIPTION
**Pull request by leewadhams**
- Pulling reviews from the repository directly, not using a QueryStore for the dashboard yet.

### CODE REVIEW

File: src/QA/QA.Web/Views/Dashboard/Index.cshtml [hunk 1]
```diff
@@ -15,29 +16,16 @@
...
-            </td>
-            <td>
-                Submitted
-            </td>
-        </tr>
+        @foreach(var item in Model)
+        {
+            <tr>
+                <td>
+                    <a asp-route="@RouteNames.Vacancy_Review_Get" asp-route-vacancyId="@item.VacancyReference">@item.Title</a>
```
Comment (sachpatel) [inline src/QA/QA.Web/Views/Dashboard/Index.cshtml]: Should this be `VacancyId`?

Comment (leewadhams) [reply src/QA/QA.Web/Views/Dashboard/Index.cshtml]: No it shouldn't

Comment (leewadhams) [reply src/QA/QA.Web/Views/Dashboard/Index.cshtml]: Fixed 👍

**Comment by leewadhams** [comment]
Closing as will be superseded by another request.

**Pull request closed without merging**